tests/unit/lambdas/daemons/test_mapper.py

```diff
@@ -118,6 +121,8 @@ def test_get_chunk_specs_ok(self, mock_dss_zarr_store, mock_zarr_group):
             self.assertEqual(chunk_spec['start_row'], i * chunk_size)
             self.assertEqual(chunk_spec['num_rows'], chunk_size)
 
+    @pytest.mark.skip(reason="Not needed while one-cell bundle assumption is in place.")
```

Comment:
Can we add a TODO here to address when assumption is no longer true?

Reply:
Yeah, filed issue #95 as well.